google-cloud-pubsub/lib/google/cloud/pubsub/subscriber/inventory.rb

```diff
@@ -115,8 +115,8 @@ def background_run
 
                 unless delay_gap.positive?
                   delay_target = nil
```

Comment:
The more I think about this, the more I wonder if this should be set to `calc_target` instead of `nil`. When we are dealing with a very short deadline, such as 10 seconds, having the call to `stream.renew_lease!` take a couple of seconds due to size of data or synchronization locking may push the `modify_ack_deadline` RPC call past the deadline. If we set `delay_target` to `calc_target` here we may mitigate that delay. Thoughts?

```suggestion
                   delay_target = calc_target
```

Reply:
I'll do this in a separate PR.